lib/util/tooltip.coffee

```diff
@@ -0,0 +1,54 @@
+# InkTooltip
+#
+# Tooltips that support arbitrary HTML content.
+#
+# new InkTooltip(parent, content, cond = -> true)
+#     Create a new InkTooltip attached to `parent` with content `content`. Will
+#     be shown on mouseover, unless `cond` evaluates to `false`.
+#
+# .show()
+#     Show the tooltip.
+#
+# .hide()
+#     Hide the tooltip.
+#
+# .destroy()
+#     Destroy the tooltip.
+
+module.exports =
+class InkTooltip
+  constructor: (@parent, content, @cond = -> true) ->
+    @view = @tooltipView content
+    document.body.appendChild @view
+    @showOnHover()
+    this
+
+  hide: -> @view.style.display = 'none'
```

Comment:
Sure, I was a bit uncomfortable with how I'm doing the `hide` and `show` right now anyways...